### PR TITLE
chore: add outline for Avatar when editable and focused

### PR DESCRIPTION
**Pull request by ozgurkececioglu**
[FX-3241]

### Description

- This PR is about adding an outline for the focused state when Avatar is editable. The requirement came from one of the other teams that Avatar should render a button instead of a div and should have an outline when it is being focused with tab key.
- I moved DropzoneSvg to reuse its settings and functionalities inside this new component (it is private to Avatar component)

### How to test

- go to [editable avatar story](https://picasso.toptal.net/fx-avatar-editable-outline/?path=/story/components-avatar--avatar) and focus on the avatars with tab key.

### Screenshots

Screenshot 2022-11-13 at 21 21 13

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests

**Breaking change**

- codemod is created and showcased in the changeset
- test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-3241]: https://toptal-core.atlassian.net/browse/FX-3241?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by TomasSlama — APPROVED** [review]
Tested in the temploy, looks good 👍

File: .changeset/seven-cooks-judge.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+'@toptal/picasso': minor
```
[the remark this answers is not in the document]
Comment (ozgurkececioglu) [reply .changeset/seven-cooks-judge.md:2]: I was thinking that I changed the underlying component so if somebody uses this component, their snapshots may have changed 🤔 that's why I go with minor. I was also in middle so let me change it to patch again 👍

[the remark this answers is not in the document]
Comment (TomasSlama) [reply .changeset/seven-cooks-judge.md:2]: maybe we can make it part of the changelog. `HTML structure changed, changes in snapshots are expected` or something like this 🤔

[the remark this answers is not in the document]
Comment (ozgurkececioglu) [reply .changeset/seven-cooks-judge.md:2]: hmm, why not 👍 good idea ❤️

**Review by mkrl — APPROVED** [review]
Looks good. Tested on temploy 👍

**Review by dmaklygin — APPROVED** [review]
Checked on temploy. Code looks good.

**Pull request merged**